docs/tutorials/working_with_large_catalogs.ipynb

```diff
@@ -2,22 +2,395 @@
  "cells": [
```

Comment:
<p>Can we change this to 'a small 5 degree radius cone from the ZTF DR? object catalog'? I think specifying the data release is good for scientists and it's better to name the catalog than quote the variable name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/236/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>